_posts/2018-11-27-pool-features.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Pool features
+author: jmmichal
+layout: post
+identifier: pool_features
+---
+
+# Introduction
+
+From the beginning of persistent memory pools existence pools have an internal state called features. It was not very useful for the end user to know what features are enabled in his pools so it was an implementation detail. The more vigilant user may notice pool features was listed in the [pmempool info][pmempool-info.1] output but not much more.
```

Comment:
Since the very first release, PMDK pools had internal feature flags. It was mostly a hidden implementation detail. The more observant users might have noticed pool features listing in `pmempool info` output, but that's about it.